src/index.html

```diff
@@ -1,22 +1,104 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400&display=swap"
+      rel="stylesheet"
+    />
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+      content="width=device-width, initial-scale=1.0"
     />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li class="nav__item">
+            <a class="nav__link1 is-active">apple</a>
```

Comment:
The 'href' attribute is missing for the 'apple' link. This could affect navigation functionality. Consider adding a valid URL or a placeholder like '#' if the link is not yet functional.